src/test/java/org/openrewrite/java/migrate/lang/NullCheckAsSwitchCaseTest.java

```diff
@@ -72,6 +75,64 @@ static String score(String obj) {
         );
     }
 
+    @Test
+    @DocumentExample
```

Comment:
Try to use just one representative example in the docs, otherwise those pages will grow long quickly.
```suggestion
```